### DISCUSSION TITLE
Add time from when to when the task has been done and a weekly review of the mixture of the tasks that I've done

### DISCUSSION THREAD

**Discussion by JelenaDragovic** [iSoron/uhabits#1649] [Feature Requests]
Hi, 
I love the app loop habit tracker! I've been using it for few months now and it's really cool!

I thought it would be just cool to track what time of the day I've decided to each of the tasks. And with it, would also be cool to have like a diagram of the week with bars (x- days and y- hours). So that i can see how i invested my time by hours a day.

Hope you find this usefull 🤗

**Comment by exanrauzer** [comment]
Hi there!

By 'what time of the day I've decided to each of the tasks' you mean the amount of time or specific timestamp?

**Comment by JelenaDragovic** [comment]
Hi,
Thank you for your fast reply. Sorry I'm a bit delayed to respond.

I mean* specific timestamp*. It would be nice if for one task would be
possible to make more than just one input as well.
Let's say my task is reading, i can put I've read 3h but I'd like to have a
track of what time of day i used for it.

I really like planning, but sometimes i find it hard  to be good at it. So
I think tracking what I've done in specific timestamps over longer period
of time (months..) could help me understand which part of days I am the
most productive for what tasks/hobbies and so.

I use google calendar to plan and I think having some *diagram illustrating
timestamps* on a weekly basis similar to it would be super cool. (I'll send
a screenshot in the attachment). I'm not sure about how could it show
the *frequency
long term*, but i guess something like a weekly window with hotspots on the
time of the day, for each day, that has been most frequent..


Hope I've explained myself good ☺️
If you have any further questions, I'd be very happy to reply and
brainstorm more of the ideas.
The app is amazing, it's just that I'm a little bit obsessed with
statistics 🤭

Kind regards,
Jelena Dragovic
